src/components/atoms/Icon/Icon.tsx

```diff
@@ -49,6 +56,12 @@ const icons: Record<IconNames, React.ComponentType<any>> = {
   images: Images,
   git: Git,
   'git-ops': GitOps,
+  'getting-started': Kubernetes,
+  unorderedListOutlined: UnorderedListOutlined,
+  rightOutlined: RightOutlined,
+  codeTwoTone: CodeTwoTone,
+  checkOutlined: CheckOutlined,
```

Comment:
I think we can still remove all of the antd icons in here and use them directly since they are not some custom icons that we added